migration/src/edited_collection.rs

```diff
@@ -0,0 +1,28 @@
+use crate::spreadsheets::SheetResult;
+use dailp::{Database};
+use anyhow::Result;
+use dailp::chapter::Chapter;
+use dailp::collection::Collection;
+
+pub async fn migrate_edited_collection(db: &Database) -> anyhow::Result<()> {
+    let res = SheetResult::from_sheet("12R07Ks8A5g2jffqoILJw7nRn6GehwNwdwosxaJBwNNM", None)
+        .await?
+        .into_collection_index(&"Test".to_string(), &4579)?;
+    let collection = res;
+
+    let collection_slug = "test_sheet".to_string();
+
+    // Insert collection into database-- hardcode CWKW stuff/move outside of this function?
+    db.upsert_collection(collection.title,collection.wordpress_menu_id, collection_slug.clone()).await?;
```

Comment:
You wrote and instantiated the `Collection` data type, why not pass that in here as the sole argument? Also, I think the `slug` should be a field on `Collection`, that matches the DB type as well.

Reply:
Right, this makes sense! And I added slug as a field-- not sure why I hadn't done that before